app/assets/stylesheets/modern-business.scss

```diff
@@ -241,3 +247,33 @@ footer {
   font-size: 40px;
   text-align: center;
 }
+
+/* Countdown Clock */
+
+#clockdiv{
+  font-family: sans-serif;
+  color: #fff;
+  display: inline-block;
+  font-weight: 50;
+  text-align: center;
+  font-size: 30px;
+}
+
+#clockdiv > div{
+  padding: 10px;
+  border-radius: 2px;
+  background: #044A8E;
```

Comment:
Color literals like `#044A8E` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#044A8E` should be written as `#044a8e`